elcid/patient_lists.py

```diff
@@ -153,25 +157,53 @@ class Bacteraemia(RfhPatientList):
     order = -10
 
 
-class PCP(RfhPatientList):
+class PCP(RfhPatientList, TaggedPatientList):
     display_name = 'PCP'
     direct_add = True
     tag = "pcp"
     template_name = 'episode_list.html'
     schema = []
 
 
-class R1(RfhPatientList):
+class R1(RfhPatientList, TaggedPatientList):
     display_name = 'R1'
     direct_add = True
     tag = "r1"
     template_name = 'episode_list.html'
     schema = []
 
 
-class LiverTransplantation(RfhPatientList):
+class LiverTransplantation(RfhPatientList, TaggedPatientList):
     display_name = 'Liver Transplantation'
     direct_add = True
     tag = "liver_transplantation"
     template_name = 'episode_list.html'
     schema = []
+
+
+class ChronicAntifungal(RfhPatientList, PatientList):
+    display_name = "Chronic Antifungal"
+    template_name = 'episode_list.html'
+    schema = []
+    is_chronic = True
```

Comment:
Do we want something like 'display_footer_buttons' etc ? 

It's not going to be just chronic as part of this release...